build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
     id "checkstyle"
     id "com.google.protobuf" version "0.9.4"
     id "org.kordamp.gradle.project-enforcer" version "0.13.0"
+    id "application"
```

Comment:
This plugin is pretty heavy. It adds distribution plugin and has some packaging stuff. I think it should already pretty easy to run the app by doing something like:
```
./gradlew build && java -jar ...
```

Reply:
ok, this works:

`./gradlew build && java -jar target/tracker-server.jar`